ui/src/components/notifications/ErrorNotification.tsx

```diff
@@ -7,7 +7,7 @@ export default function ErrorNotification() {
 
   return (
     <Transition show={error !== null}>
-      <div className="max-w-s fixed bottom-0 right-2 z-50 m-4">
+      <div className="max-w-s fixed bottom-0 right-2 z-50 m-4 w-1/3">
```

Comment:
does this look ok on other pages? that cause errors?

Reply:
actually, what if instead we do `max-w-1/3`? that way for notifications with shorter error messages it still will fit those without being too large? would that work here?

Reply:
@markphelps That works. I tested this on other errors yes! but yeah i think having it `max` makes sense.

Reply:
@markphelps Hmm i tried this `max-w-1/3` and I am not sure if that is a proper setting with Tailwind. It seemed as though this error displayed above was one of the more larger ones throughout the app, so I picked a safe value.